ui/component/publishForm/view.jsx

```diff
@@ -387,6 +405,12 @@ function PublishForm(props: Props) {
 
     const isValid = uri && isURIValid(uri);
     if (uri && isValid && checkAvailability && name) {
+      const otherUriToResolve = alternateCapitalizeString(uri);
+      if (otherUriToResolve) {
+        resolveUri(otherUriToResolve);
+        checkAvailability(alternateCapitalizeString(name));
+      }
+
       resolveUri(uri);
       checkAvailability(name);
```

Comment:
I thought of something wanted to try it out to confirm, but crashed:

![image](https://user-images.githubusercontent.com/64950861/162167634-b048b84f-17b8-4bcf-a874-7b0b7a855a68.png)

Anyway, wouldn't the `toLowerCase` method require us to pre-resolve/fetch all of our claims, which we shouldn't do since wallets could be big?

Reply:
we should also be able to do a claim list with the --name param, which is not case sensitive i think (and if so, pass lowercase version).

Reply:
Yeah, that is what's happening in prod now, but I think it is case-sensitive.

I have these 2 in my wallet
```
lbry://@ip-chans-6#9/hello-baby#d
lbry://@ip-chans-6#9/hello-BABY#c
```
But a query with `name: "hello-baby"` only returned the first one.

Reply:
ewwwww